Inspector/js/inspector.js

```diff
@@ -56,6 +56,7 @@ class Inspector extends React.Component {
         {this.renderField('Rect', attributes.rect)}
         {this.renderField('isEnabled', boolToString(attributes.isEnabled))}
         {this.renderField('isVisible', boolToString(attributes.isVisible))}
+        {this.renderField('isFocused', boolToString(attributes.isFocused))}
```

Comment:
does it still work with iOS? should we have some condition there?

Reply:
In iOS case, it always shows `No`, but it does not need for them. Changed to return `nil` (the section disappears) if the attribute does not exist.